dev/run_dev.R

```diff
@@ -1,3 +1,6 @@
+# Sass code compilation
+sass::sass(input = sass::sass_file("inst/app/www/timevis.sass"), output = "inst/app/www/timevis.css", cache = NULL)
```

Comment:
I'm a newbie to {sass} but this should probably be copied over to `app.R` too.

Reply:
It doesn't need to be because it creates the `timevis.css` file. If we are using the workflow of editing then running `golem::run_dev()`, any changes made to the `timevis.sass` file will be implemented to `timevis.css`.

Reply:
> 1. there appears to be some non-existent events & dates popping up on initiation of the plot. Below, I chose the first patient (01-701-1015) and normally, the green dot with the start next to it is the first event for this patient. There should be no adverse events (or anything else) for this matter before this dot, since it is the disease onset date, which is waaay before the

I'm having trouble reproducing this issue. Was there anything else that you did or occurred?